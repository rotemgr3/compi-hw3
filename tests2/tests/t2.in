
//piazza 65 - should be ok with 1 scope for if +
void main() {
	int a = 5;
	int d = 7;
	if (a > d)
	int c = 3;
}