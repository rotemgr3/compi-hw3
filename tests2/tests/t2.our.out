---end scope---
c INT 2
---end scope---
a INT 0
d INT 1
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
main ()->VOID 0
